config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_for :users
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root "splash#index"
+
+  resources :users
+
+  resources :groups do
+    resources :expenses
+  end
```

Comment:
- [OPTIONAL] It would be nice to use "only" and "except" options to add only necessary routes in case maybe there are some routes that are not used. 👍🏻